easybuild/framework/easyconfig/tools.py

```diff
@@ -292,7 +307,7 @@ def det_easyconfig_paths(orig_paths, from_pr=None, easyconfigs_pkg_paths=None):
     return [(ec_file, False) for ec_file in ec_files]
 
 
-def parse_easyconfigs(paths):
+def parse_easyconfigs(paths, validate=True):
```

Comment:
use `**kwargs`, if you are going to pass it as-is to `process_easyconfig`  (and add it to it the docstring)